include/psa/crypto_extra.h

```diff
@@ -434,6 +434,9 @@ psa_status_t mbedtls_psa_inject_entropy(const uint8_t *seed,
 #define PSA_PAKE_OPERATION_STAGE_COLLECT_INPUTS 1
 #define PSA_PAKE_OPERATION_STAGE_COMPUTATION 2
 
+/** JPAKE operation stages. */
```

Comment:
It is temporary, but I think it is useful to have it for now. Were do you suggest to move it?

Reply:
I'd say being temporary is an additional reason not to be in a public header :) I'm not sure where to move it, though.

I think `ssl_tls.c` should have its own copy: it's using those strings because there's a TLS RFC (well, draft) that says so, regardless of what the PSA implementation supports or not.

For the other users (`psa_crypto.c`, `test_suite_psa_crypto_driver_wrappers.function` and `test_suite_psa_crypto_pake.function`) I'm not sure. I'm tempted to say each of them could have its own copy: it's not like we're going to change those values, and we already have copies of those strings in `test_suite_psa_crypo_pake.data` anyway.

Wdyt?

Reply:
Ok I'll move this to source files.